packages/engine/lib/stateful/src/message/arrow/array.rs

```diff
@@ -35,7 +35,7 @@ pub struct AgentMessage {
     r#data: Option<String>,
 }
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
For what this `Clone` is required? I don't mind if it's required, but if it can be avoided, this would be great.

Reply:
Gotcha, that's unfortunate 🙁